packages/multichain-network-controller/src/MultichainNetworkService/MultichainNetworkService.test.ts

```diff
@@ -0,0 +1,127 @@
+import type { CaipAccountId } from '@metamask/utils';
+
+import { MultichainNetworkService } from './MultichainNetworkService';
+import {
+  type ActiveNetworksResponse,
+  MULTICHAIN_ACCOUNTS_CLIENT_HEADER,
+  MULTICHAIN_ACCOUNTS_CLIENT_ID,
+  MULTICHAIN_ACCOUNTS_BASE_URL,
+} from '../api/accounts-api';
+
+describe('MultichainNetworkService', () => {
+  const mockFetch = jest.fn();
+  const validAccountIds: CaipAccountId[] = [
+    'eip155:1:0x1234567890123456789012345678901234567890' as CaipAccountId,
+    'solana:1:EPjFWdd5AufqSSqeM2qN1xzybapC8G4wEGGkZwyTDt1v' as CaipAccountId,
```

Comment:
We shouldn't need those type-casts:

```suggestion
    'eip155:1:0x1234567890123456789012345678901234567890',
    'solana:1:EPjFWdd5AufqSSqeM2qN1xzybapC8G4wEGGkZwyTDt1v',
```